stablehlo/reference/Element.h

```diff
@@ -75,6 +75,12 @@ class Element {
   /// complex type.
   std::complex<APFloat> getComplexValue() const;
 
+  /// Returns the implementation-defined bits of the underlying value.
+  APInt toBits() const;
+
+  /// Creates an Element from implementation-defined bits.
+  static Element fromBits(Type type, APInt bits);
```

Comment:
Similarly to the PR for convert, when reviewing this PR I noticed a high degree of duplication, so I tried to come up with orthogonal primitives that would then be combined with each other (rather than copypasted into multiple places). `toBits` and `fromBits` did the trick - the code has become much shorter and much simpler.